app/database_etl/postgres_tables_handler/postgres_utils.py

```diff
@@ -74,7 +74,29 @@ def get_filter_static_options() -> Dict[str, List[str]]:
             "ELISA",
             "LFIA",
             "Other"
-        ]
+        ],
+        "genpop": [
```

Comment:
Just want to note that the only reason some of these filter options are hardcoded is because they need to be shown in a certain order on serotracker.com.

That being said, do we ever want to show these filter options on the site / do they ever need to be ordered in a specific way? If not we could just use a select distinct query in `get_all_filter_options` below to get the options for these filters. Example code: 

```
query = session.query(distinct(ResearchSource.genpop))
results = [q[0] for q in query if q[0] is not None]
# sort countries in alpha order
options["genpop"] = sorted(results)
```

Reply:
This is a great idea. No reason for us to hardcode this. Will modify to generate filter options dynamically for these variables.